packages/flutter_test/lib/src/binding.dart

```diff
@@ -1630,9 +1630,8 @@ enum LiveTestWidgetsFlutterBindingFramePolicy {
   benchmarkLive,
 }
 
-/// A variant of [TestWidgetsFlutterBinding] for executing tests in
-/// the `flutter run` environment, on a device. This is intended to
-/// allow interactive test development.
+/// A variant of [TestWidgetsFlutterBinding] for executing tests
+/// on a device. This is intended to allow interactive test development.
```

Comment:
this should also continue to mention `flutter run` as a typical way.

Reply:
done